scss/styles.scss

```diff
@@ -0,0 +1,296 @@
+@import "_reset";
+@import "_mixins";
+@import "_fonts";
+
+//variables
+$main-gray: #626262;
+$support_gray: #f3f3f3;
+$aside: #b4ad9e;
+
+
+$firstBreakPoint: 600px;
+$secondBreakPoint: 900px;
+$thirdBreakPoint: 1200px;
+
+
+
+
+body{
+  font-family: Playfair, Arial, sans-serif;
+  font-size: 16px;
+}
+.main-content-wrapper{
+  padding-top: 56px;
+}
+
+.wrapper{
+  max-width: 1200px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+/**********
+HEADER styles
+***********/
+
+.dropdown{
+  position: absolute;
+  top: -500px;
+  right: -500px;
+  transition: .4s;
+  @include mQ($secondBreakPoint) {
+    position: static;
+
+  }
+}
+
+.header-wrapper{
+  background: white;
+  position: fixed;
+  display:flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: .5em;
+  @include mQ($thirdBreakPoint){
+    position:static;
+  }
+  .main-logo{
+    display: block;
+    max-width:119px;
```

Comment:
To są generalnie `magic-numbers` (podobnie z pozostałymi wartościami, np. `padding-top: 56px`. Zawsze jak chcesz napisać coś takiego to zastanów się czy jesteś to w stanie jakoś wyliczyć? Jeśli nie to i tak powinieneś to wyrzucić do zmiennych jako np. `$main-logo-max-width`, żeby pozostali `devowie mieli te wartości w jednym miejscu w zmiennych